### PR TITLE
raidboss: TOP P5 Sigma Staff/Feet Call

### PR DESCRIPTION
**Pull request by Legends0**
The logs seem a bit strange on this one. Omega-M which casts Discharge, is seen as casting the spell but Omega-F is what readies it. Unsure if this is capturing the correct npc model.

### CODE REVIEW

**Comment by Legends0** [comment]
Last I tested this, it's throwing an 'error stopped' like message when the trigger activates in sigma but trigger actives fine on the p2 Discharger event. I am not quite sure what is causing the error. It's likely happening in either the promise or in the data.combatantData.pop() during runtime.

**Comment by quisquous** [comment]
Hmm.  I don't think `data.combatantData.pop()` could go wrong unless `data.combatant` was undefined.  "Error stopped" sounds a lot more like #5153, where the pull is over before the delayed trigger fires.

File: ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts [hunk 1]
```diff
@@ -1335,6 +1335,43 @@ const triggerSet: TriggerSet<Data> = {
         return { infoText: output.noDebuff!() };
       },
     },
+    {
+      id: 'TOP Sigma Superliminal/Blizzard',
+      // Omega-M casts Superliminal/Blizzard
+      // Track from Discharger (7B2E)
+      type: 'Ability',
+      netRegex: { id: '7B2E', source: 'Omega' },
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts:1343]:
```suggestion
      netRegex: { id: '7B2E', source: 'Omega-M' },
```

Maybe this?

Comment (Legends0) [reply ui/raidboss/data/06-ew/ultimate/the_omega_protocol.ts:1343]: I think you're correct here. The P2 is just Omega, but P5 here is Omega-M and the P2 and P5 npc are different sourceIDs.

P2:
```
|4000ED62|Omega|7B2E|Discharger|
```

P5 in same pull:
```
|4000EEB8|Omega-M|7B2E|Discharger|
|4000EEB8|Omega-M|7B2A|Superliminal Steel|4000EEB8|Omega-M|1.200|90.00|100.00|0.00|
```

**Comment by Legends0** [comment]
I dropped the delay to 6.2, I swear I had tested with 6 on my part and it did not work, but it may work at 6.2 if others seeing it less than 6 from the ability going off in the log.

**Pull request merged**